### ISSUE TITLE
Edit quota button for a batch action should not be displayed if at least one user has not logged on

### ISSUE THREAD

**Issue by ScharfViktor** [owncloud/web#8532]
ownCloud Web UI 7.0.0-rc.16 

Precondition:
- start ocis with demo users
- log in as einstein

Steps:
- log in as admin
- open Administration settings->User tab 

- check checkbox einstein -> `edit quota` button appears - it's correct
- check checkbox katherina  -> `edit quota` button stills visible -not correct

Actual: `edit quota` button stills visible. after clicking by `edit quota` button -> web sends only one request for einstein
IMHO it is not correct. because the admin thinks that he has already set up a quota for Katherina, which is not the case

https://user-images.githubusercontent.com/84779829/222715780-af37eb51-2ec6-410a-b877-e6b30a3dfad5.mov

**Comment by kulmann** [comment]
It's implemented as specified by @tbsbdr and myself, but I also see possible confusion here. I don't agree that the edit quota button should be hidden (and actually at the moment we only know the login state per user, not in the list, so we couldn't even build that). But we should at least print some warning that some users didn't receive quota by this action... @tbsbdr what do you think?

**Comment by tbsbdr** [comment]
hm.. correct, the actual problem is the "user needs to have logged in once." issue and now we try to mitigate it.
tbh I'd favour to find a fix for the root cause of this problem 😈 maybe not now and maybe only if we get feedback that it really hurts ( @ChrisEdS already reported that this is annoying for him)

**Alternatives** could be

1) print a general warning as you said @kulmann 
    - pro: simple
    - con: you still don't know which accounts did not receive quota - admin is not in control in the overview
2) show quota as column in listing
    - pro: admin gets immediate, detailed feedback about the quota status of each user
    - con: expensive in terms of performance ?

@kulmann lets discuss next week what we favour. maybe there is also an option 3 or 4 ....

**Comment by kulmann** [comment]
Of course, ideal solution would be to be able to show the user quota in the list and a `-` or similar for users that never logged in. Then we could apply our pattern of suffixing the number of users a request will be sent for in the batch action button. But that's not an option performance wise. We can't expand the personal drives in the user listing via graph api. Far too expensive.

The general warning would be a mitigation. And because of that I wouldn't invest more than just showing generic text.

**Comment by tbsbdr** [comment]
> And because of that I wouldn't invest more than just showing generic text.

👍 yes, lets go with this approach

**Comment by kulmann** [comment]
`Quota will only be applied to users who logged in at least once.`

**Comment by JanAckermann** [comment]
Decided to also show affected users in a ct helper
